whois-update/src/main/java/net/ripe/db/whois/update/domain/UpdateMessages.java

```diff
@@ -616,6 +616,10 @@ public static Message statusCannotBeRemoved() {
         return new Message(Type.WARNING, "\"status:\" attribute cannot be removed");
     }
 
+    public static Message emailCanNotBeSent(final String email, final String emailStatus) {
+        return new Message(Type.WARNING, "Not sending notification to %s because it is %s.", email, emailStatus);
```

Comment:
I also think the error message should be short? Something like <email> is unsubscribed or <email> is undeliverable else it can become a really long warnings ? But feel free to Ignore if you think otherwise

Reply:
I am fine putting EmailStatus in lowercase instead of uppercase.